src/telemetry/surveys/awareness.ts

```diff
@@ -24,7 +23,6 @@ async function isEligible(): Promise<boolean> {
     const overallActivity = ext.activityMeasurementService.getActivityMeasurement('overall');
     const noEditActivity = ext.activityMeasurementService.getActivityMeasurement('overallnoedit');
 
-    return (vscode.env.language === 'en' || vscode.env.language.startsWith('en-')) &&

```

Comment:
TODO: this change essentially means we only show to Docker extension 1.4.0, assuming we also turn on the language filter on the awareness flight. Is that OK?

Reply:
I'd think so, given how quickly people have been upgrading the extension in the past

Reply:
Talked to @BigMorty, he's OK with locking this experiment to 1.4.0+, thus doing this change is OK.